### PR TITLE
Removed workaround for mouseDown event in jest tests

### PR DESCRIPTION
**Pull request by vhuseinova-msft**
# What

- Removed workaround for mouseDown event in jest tests
- Removed duplicated strict mode from Calling sample app

# Why

# How Tested

Ran tests

# Process & policy checklist

- [x] I have updated the project documentation to reflect my changes if necessary.
- [x] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

File: samples/Calling/src/app/App.tsx [hunk 1]
```diff
@@ -204,18 +204,16 @@ const App = (): JSX.Element => {
         return <Spinner label={'Getting user credentials from server'} ariaLive="assertive" labelPosition="top" />;
       }
       return (
-        <React.StrictMode>
```
Comment (JamesBurnside) [inline samples/Calling/src/app/App.tsx:207]: keep strict mode?

**Review by JamesBurnside — APPROVED** [review]

**Review by emlynmac — APPROVED** [review]
We should leave strict mode enabled but good to remove the workaround

File: samples/Calling/src/app/App.tsx (hunk 1 above)
Comment (vhuseinova-msft) [reply samples/Calling/src/app/App.tsx:207]: @JamesBurnside, @emlynmac it is enabled in `index.tsx` for all Sample apps. The one that is deleted in this PR, is additional call in `App.tsx` file which is not needed anymore.

Comment (JamesBurnside) [reply samples/Calling/src/app/App.tsx:207]: Ah! 💖 the extra cleanup!

**Pull request merged**